smart-contracts/faucets.md

```diff
@@ -1 +1,44 @@
-TODO
\ No newline at end of file
+---
+title: Faucets
+description: Learn how to obtain test tokens from Polkadot faucets for development and testing purposes across different networks.
+categories: Smart Contracts
+---
+
+# Faucets
+
+Test tokens allow you to experiment with smart contracts, test transactions, and validate your dApp functionality without using real cryptocurrency. Polkadot has a reliable faucet that distributes free TestNet tokens to developers for testing purposes.
+
+## Faucet Basics
+
+Faucets are services that distribute small amounts of cryptocurrency or tokens for free, typically on TestNet networks. They serve as the primary method for developers to obtain the necessary tokens to:
+
+- Execute transactions on TestNet networks.
+- Pay for gas fees when deploying smart contracts.
+- Test dApp functionality before MainNet deployment.
+- Experiment with blockchain interactions without financial risk.
+
+## Get Test Tokens
+
+For Polkadot Hub TestNet, you can use the [Polkadot Faucet](https://faucet.polkadot.io/?parachain=1111){target=\_blank} to obtain test tokens. Here's how to do it:
+
+1. Navigate to the [Polkadot Faucet](https://faucet.polkadot.io/?parachain=1111){target=\_blank}. If the desired network is not already selected, choose it from the Network drop-down. This example uses the Polkadot Hub TestNet.
+
+2. Copy your address linked to the TestNet and paste it into the designated field.
+
+    ![](/images/smart-contracts/faucets/faucets-1.webp)
+
+3. Click the **Get Some PASs** button to request free test PAS tokens. These tokens will be sent to your wallet shortly.
+
+    ![](/images/smart-contracts/faucets/faucets-2.webp)
+
+## Things to Consider
+
+!!! info "Rate Limiting"
+    Faucets typically implement rate limiting to prevent abuse. You may need to wait between requests if you've recently obtained tokens from the same faucet.
+
+!!! warning "Network Compatibility"
+    Ensure your wallet is connected to the correct network (Polkadot Hub TestNet) before requesting tokens. Tokens sent to addresses on different networks will not be accessible.
```

Comment:
Ah I'm not a fan of having a section of just admonitions, I don't think it looks right.

I think the Network Compatibility warning should be maybe be it's own step  under Get Test Tokens or at least as an admonition under a relevant step

I think the Rate Limiting info could be at the end of the Get Test Tokens section.